jtlibrary/python/jtmodules/src/jtmodules/separate_clumps.py

```diff
@@ -70,15 +70,18 @@ def main(mask, intensity_image, min_area, max_area,
         after the cuts were performed (helps to see why some objects were not
         cut, especially if there are complicated clumps that require multiple
         cuts). Defaults to false, thus showing the values in the original image
+    trimming: bool
+        some cuts may create a tiny third object. If this boolean is true, 
+        tertiary objects < trimming_threshold (10) pixels will be removed
 
     Returns
     -------
     jtmodules.separate_clumps.Output
     '''
 
-    separated_mask = separate_clumped_objects(
+    separated_label_image = separate_clumped_objects(
         mask, min_cut_area, min_area, max_area,
-        max_circularity, max_convexity
+        max_circularity, max_convexity, allow_trimming = trimming
```

Comment:
Python's stylistic convention is to avoid spaces around `=` when used to pass named arguments to functions.